packages/server/lib/browsers/chrome.ts

```diff
@@ -608,9 +612,11 @@ export = {
     const browserCriClient = this._getBrowserCriClient()
 
     // Handle chrome tab crashes.
-    pageCriClient.on('Inspector.targetCrashed', () => {
+    pageCriClient.on('Inspector.targetCrashed', async () => {
       const err = errors.get('RENDERER_CRASHED')
 
+      await memory.endProfiling()
```

Comment:
Here is another place where we handle the browser crashing (the whole thing, not just the tab), should we end profiling here too?

https://github.com/cypress-io/cypress/blob/e973fa3901bbf22cdab323b91c19f11816fbefcd/packages/server/lib/browsers/index.ts#L182-L194

Reply:
Updated to listen for the `exit` event and end profiling there. [d0daff0](https://github.com/cypress-io/cypress/pull/25462/commits/d0daff04839e9229a9bb9a1f250473a46a4f07eb)